.build/package.json

```diff
@@ -56,15 +56,16 @@
     "start": "webpack-cli serve",
     "start:dev": "webpack serve --hot --color --progress --config webpack.dev.js",
     "clean": "rimraf dist tmp && gh-pages-clean",
-    "generate:attributes": "node ./utils/attributes.js",
-    "generate:modular-docs-sources": "node ./utils/modular-docs.js",
     "commitandpush:modular-docs": "gh-pages -d ./tmp/post-splitter -b modular-docs",
     "publish:modular-docs": "npm run generate:modular-docs-sources && npm run commitandpush:modular-docs",
     "fetch:templates": "node ./utils/templates.js",
+    "fetch:splitter": "mvn -f utils/pom.xml package",
     "build:generated-assemblies": "asciidoctor --failure-level=WARN  --template-dir=templates --destination-dir=tmp/generated/assemblies tmp/post-splitter/assemblies/assembly-*.adoc",
-    "build:generated-modules": "asciidoctor --failure-level=WARN --trace --template-dir=templates --destination-dir=tmp/generated/modules tmp/post-splitter/modules/*/*.adoc",
+    "build:generated-modules": "asciidoctor --failure-level=WARN --trace --template-dir=templates --destination-dir=tmp/generated/modules tmp/post-splitter/modules/*/*.adoc","generate:attributes": "node ./utils/attributes.js",
```

Comment:
Maybe I'm missing something, but it looks like you're a step per line for readablity. This one should probably follow suit.

Reply:
Thanks - yes, I just realised that. I also realised I can get rid of the `pom.xml` and just use the mvn command line to get the dependency directly.